labcodes/confetti-storybook
# CSYS-274: Fix stories structure

**Link to task:**
https://labcodes.atlassian.net/browse/CSYS-274

**Staging app:**
https://fix-stories-structure--confetti-storybook.netlify.app/

**How to test:**
Access the staging app and check the structure of stories on the left sidebar. Verify if that's in accordance with expected. Also check that the description of props are show in 'Controls' section of each story.

**Description of your solution:**
This PR fix the stories structure, according to the description in the JIRA card (see the task description for details).
Notes:
- Tag structure partially applied. The files are still in branch to be merged.
- 'Email Input' and 'Password Input' are placed into 'Data Inputs/Text Iput' structure.
